tools/readme_generator/translations/ca/LC_MESSAGES/messages.po

```diff
@@ -35,21 +35,21 @@ msgid ""
 "template'](https://github.com/YunoHost/example_ynh/generate) button on "
 "the Github repo."
 msgstr ""
-"Copia aquesta aplicació abans de treballar-hi, utilitza el botó ['Utilitza "
-"aquesta plantilla'](https://github.com/YunoHost/example_ynh/generate) al "
+"Copia aquesta aplicació abans de treballar-hi, utilitza el botó [«Utilitza "
+"aquesta plantilla»](https://github.com/YunoHost/example_ynh/generate) al "
 "repositori de Github."
 
 #: templates/README.md.j2:5
 msgid "Edit the `manifest.json` with app specific info."
-msgstr "Edita el `manifest.json`amb la informació específica de l'aplicació."
+msgstr "Edita el «manifest.json» amb la informació específica de l'aplicació."
 
 #: templates/README.md.j2:6
 msgid ""
 "Edit the `install`, `upgrade`, `remove`, `backup`, and `restore` scripts,"
 " and any relevant conf files in `conf/`."
 msgstr ""
-"Edita els scripts `install`, `upgrade`, `remove`, `backup`, i `restore`, i "
-"tots els arxius rellevants de `conf/`."
+"Edita els scripts «install», «upgrade», «remove», «backup», i «restore», i "
+"tots els arxius rellevants de «conf/»."
```

Comment:
uh no :<

Reply:
argh, we need to add flags to tell weblate to stop pretending he is intelligent x_x 

https://docs.weblate.org/en/latest/user/checks.html#autofix
https://docs.weblate.org/en/latest/admin/checks.html#custom-checks

Reply:
i manually reverted these edits in weblate